library/psa_crypto_slot_management.h

```diff
@@ -203,12 +203,16 @@ psa_status_t psa_validate_key_location( psa_key_lifetime_t lifetime,
 /** Validate the persistence of a key.
  *
  * \param[in] lifetime  The key lifetime attribute.
+ * \param[in] key_id    The key identifier.
  *
  * \retval #PSA_SUCCESS
+ * \retval #PSA_ERROR_INVALID_HANDLE
```

Comment:
sorry, do I miss something? didn't see this return value in the function implementation.